pages.de/linux/aplay.md

```diff
@@ -0,0 +1,16 @@
+# aplay
+
+> Commandline Musik Player für den ALSA Soundkarten Treiber.
```

Comment:
I can't find a better translation of `command-line` myself, but I think we should at least keep the dash
```suggestion
> Command-line Musik Player für den ALSA Soundkarten-Treiber.
```